test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/AtomicCryptoTransferHTSSuite.java

```diff
@@ -0,0 +1,1308 @@
+                                                                    .build(),
+                                                            EMPTY_TUPLE_ARRAY)
+                                                    .payingWith(GENESIS)
+                                                    .via(cryptoTransferTxn)
+                                                    .gas(GAS_TO_OFFER),
+                                            // Sender does not have key
+                                            contractCall(
+                                                            CONTRACT,
+                                                            TRANSFER_MULTIPLE_TOKENS,
+                                                            transferList()
+                                                                    .withAccountAmounts(
+                                                                            accountAmount(
+                                                                                    sender2,
+                                                                                    -amountToBeSent,
+                                                                                    false),
+                                                                            accountAmount(
+                                                                                    receiver,
+                                                                                    amountToBeSent,
+                                                                                    false))
+                                                                    .build(),
+                                                            EMPTY_TUPLE_ARRAY)
+                                                    .payingWith(GENESIS)
+                                                    .via(cryptoTransferRevertNoKeyTxn)
+                                                    .gas(GAS_TO_OFFER)
+                                                    .hasKnownStatus(CONTRACT_REVERT_EXECUTED),
+                                            // Sender balance is too low
+                                            contractCall(
+                                                            CONTRACT,
+                                                            TRANSFER_MULTIPLE_TOKENS,
+                                                            transferList()
+                                                                    .withAccountAmounts(
+                                                                            accountAmount(
+                                                                                    sender,
+                                                                                    -1000
+                                                                                            * ONE_HUNDRED_HBARS,
```

Comment:
Here and below - is this the intended formatting? "<amount> <line break> <* units>"

Reply:
I believe spotlessApply did that.  We are not able to check in code unless it is passed through spotless and sometimes it does strange things.